lib/cisco_node_utils/fabricpath_global.rb

```diff
@@ -0,0 +1,439 @@
+                   state: '') if curr == :uninstalled
+        config_set('fabricpath', 'feature', state: '')
+      when :disabled
+        config_set('fabricpath', 'feature', state: 'no') if curr == :enabled
+        return
+      when :installed
+        config_set('fabricpath', 'feature_install',
+                   state: '') if curr == :uninstalled
+      when :uninstalled
+        config_set('fabricpath', 'feature', state: 'no') if curr == :enabled
+        config_set('fabricpath', 'feature_install', state: 'no')
+      end
+    rescue Cisco::CliError => e
+      raise "[#{@name}] '#{e.command}' : #{e.clierror}"
+    end
+
+    def create
+      FabricpathGlobal.fabricpath_feature_set(:enabled) unless
+      :enabled == FabricpathGlobal.fabricpath_feature
+    end
+
+    def destroy
+      @name = nil
+      FabricpathGlobal.fabricpath_feature_set(:disabled)
+    end
+
+    def my_munge(property, set_val)
+      val = config_get_default('fabricpath', property)
+      case property
+      when /loadbalance_algorithm/
+        if (val == 'source-destination') &&
+           (set_val == 'symmetric' || set_val == 'xor')
+          val
+        else
+          set_val
+        end
+      when /loadbalance_.*_rotate/
+        if val == false || set_val == ''
+          ''
+        else
+          "rotate-amount 0x#{set_val.to_s(16)}"
+        end
+      else
+        set_val
+      end
+    end
+
+    ########################################################
+    #                      PROPERTIES                      #
+    ########################################################
+
+    def aggregate_multicast_routes
+      config_get('fabricpath', 'aggregate_multicast_routes')
+    end
+
+    def aggregate_multicast_routes=(val)
+      @set_params = {}
+      @set_params[:state] = val ? '' : 'no'
+      config_set('fabricpath', 'aggregate_multicast_routes', @set_params)
+    rescue Cisco::CliError => e
+      raise "[Setting agg-routes #{val}] '#{e.command}' : #{e.clierror}"
+    end
+
+    def default_aggregate_multicast_routes
+      config_get_default('fabricpath', 'aggregate_multicast_routes')
+    end
+
+    def allocate_delay
+      config_get('fabricpath', 'allocate_delay')
+    end
+
+    def allocate_delay=(val)
+      @set_params = {}
+      @set_params[:state] = val ? '' : 'no'
+      @set_params[:delay] = val ? val : ''
+      config_set('fabricpath', 'allocate_delay', @set_params)
+    rescue Cisco::CliError => e
+      raise "[Setting allocate-delay #{val}] '#{e.command}' : #{e.clierror}"
+    end
+
+    def default_allocate_delay
+      config_get_default('fabricpath', 'allocate_delay')
+    end
+
+    def auto_switch_id
+      config_get('fabricpath', 'auto_switch_id')
+    end
+
+    def graceful_merge
+      graceful_merge_conf = config_get('fabricpath', 'graceful_merge')
+      # opposite meaning with the cli
+      return true if graceful_merge_conf.nil?
+      graceful_merge_conf ? false : true
```

Comment:
Just replace entire method with:

```
def graceful_merge
  # invert result because graceful-merge nvgens with 'disable' keyword
  not config_get('fabricpath', 'graceful_merge')
end
```